README.md

```diff
@@ -0,0 +1,352 @@
+uhashring
+=========
```

Comment:
Can you please convert those (and the --- ones) to # anchors please?

Reply:
Sorry to have missed that, I fixed it! :+1: